### ISSUE TITLE
Nullpointer exception with CableMultipart.tickPower

### ISSUE THREAD

**Issue by leagris** [TechReborn/TechReborn#1125]
### Versions

- IndustrialCraft-2: 2.7.60-ex111
- MCMultiPart: 2.0.0
- Minecraft Forge 1.11.2: 13.20.0.2311
- RebornCore 1.11.2: 2.16.1.115
- TechReborn 1.11.2: 2.3.3.174

### Minecraft Crash Report

[Link](https://pastebin.com/7QAPQVKH)
```
Thread: Client thread
Stacktrace:
    at techreborn.parts.powerCables.CableMultipart.tickPower(CableMultipart.java:292)
    at techreborn.parts.powerCables.CableMultipart.func_73660_a(CableMultipart.java:278)
    at reborncore.mcmultipart.block.TileMultipartContainer$Ticking.func_73660_a(TileMultipartContainer.java:312)
 
-- Block entity being ticked --
Details:
    Name: reborncore.mcmultipart:multipart.ticking // reborncore.mcmultipart.block.TileMultipartContainer.Ticking
    Block type: ID #3487 (tile.null // reborncore.mcmultipart.block.BlockMultipartContainer)
    Block data value: 0 / 0x0 / 0b0000
    Block location: World: (-1792,71,509), Chunk: (at 0,4,13 in -112,31; contains blocks -1792,0,496 to -1777,255,511), Region: (-4,0; contains chunks -128,0 to -97,31, blocks -2048,0,0 to -1537,255,511)
    Actual block type: ID #3487 (tile.null // reborncore.mcmultipart.block.BlockMultipartContainer)
    Actual block data value: 0 / 0x0 / 0b0000
Stacktrace:
    at net.minecraft.world.World.func_72939_s(World.java:1802)
```

**Comment by modmuss50** [comment]
Does this happen all the time? This wont be able to happen in 1.12 as we fully rewrote the cables.

**Comment by leagris** [comment]
It is random crash.

Now I have a bad feeling you will only fix it for 1.12.
It is not like magically all other mods we use are ported to 1.12.
Would you be kind looking to fix it in 1.11.2?

**Comment by drcrazy** [comment]
Not reproducible in 1.12
